package.json

```diff
@@ -31,29 +35,35 @@
       "no-param-reassign": 0
     }
   },
-  "repository": "github:hydra-newmedia/hapi-sentry",
+  "engines": {
+    "node": ">=8.9.0"
+  },
+  "repository": "github:space-between/hapi-sentry",
   "keywords": [
     "hapi",
     "sentry",
     "plugin"
   ],
-  "author": "Christian Hotz <hotz@hydra-newmedia.com>",
+  "author": "Thomas Purchas<thomas.purchas@hrgo.co.uk>",
```

Comment:
Please revert this.
You are more than welcome to add yourself to the `contributors` field tho: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#people-fields-author-contributors

We can maybe think about it again if we merge all of your PRs – at that point it's probably more of your code than ours. :sweat_smile:

Reply:
@fiws would you be ok with me moving everyone into the contributors field and removing the author field completely? We can take the opportunity to add yourself in addition to @guischdi